exporters/trace/jaeger/jaeger.go

```diff
@@ -52,7 +53,14 @@ type options struct {
 	// BatchMaxCount defines the maximum number of spans sent in one batch
 	BatchMaxCount int
 
-	Config *sdktrace.Config
+	// DefaultSampler is the default sampler used when creating new spans.
+	DefaultSampler sdktrace.Sampler
+
+	// SpanLimits used to limit the number of attributes, events and links to a span.
+	SpanLimits sdktrace.SpanLimits
+
+	// Resource contains attributes representing an entity that produces telemetry.
+	Resource *resource.Resource
```

Comment:
Replicating the `sdktrace.TracerProviderOption`s here makes me somewhat uneasy.  Following this pattern would require that they be duplicated in every exporter and that any new `TracerProviderOption`s be added in multiple places.  I wonder if instead a `WithSDKOptions(opts ...sdktrace.TracerProviderOption)` would be better.  Then we could store a `[]sdktrace.TracerProviderOption` in the exporter config and append `sdktrace.WithSyncer(exporter)` to it just before calling `sdktrace.NewTracerProvider(pOpts...)`

Reply:
Done.